docs/content/en/docs/workflows/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Common Workflows"
-linkTitle: "Common Workflows"
+title: "Working With Skaffold"
+linkTitle: "Working With Skaffold"
 weight: 3
```

Comment:
i just realized index and ci-cd has same wt.
Can we increment the wt by 10 here in all the files?

Reply:
hmm, not sure I see that. `_index.md` has weight 3, and `ci-cd.md` has weight 20?

I fixed up the weights anyway